edgexfoundry/edgex-compose
# refactor!: Rework service names to conform to consistent naming

BREAKING CHANGE: Many sevice names have changed in order to conform with core-, support-, app- and device- naming standard. Impact should be minimal unless scripts have references to the old service names, rather than container names. Container name havent chnaged except for app-functional-tests' which is now edgex-app-functional-tests

closes #231
Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
run `make run ds-virtual` 
Verify all service bootstrap and data is flowing from Device Virtual to App Rules Engine
